tap_mysql/tap.py

```diff
@@ -116,6 +116,17 @@ def __init__(
                 "for more information"
             ),
         ),
+        th.Property(
+            "skip_ordering",
+            th.BooleanType,
+            default=False,
+            description=(
+                "If true, disables ORDER BY when fetching data. "
+                "It can be useful when you have huge json columns inside your database and "
+                "ordering them throws memory related issues. "
+                "This may negatively affect incremental syncs and should be used with caution."
+            ),
+        ),
```

Comment:
I wonder if this should be an enum rather than a boolean. Initial options would be `replication_key` and `none`, but we could expand from there in the future. What do you think?